specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-11-01/managedClusters.json

```diff
@@ -3413,6 +3421,216 @@
         "modelAsString": true
       },
       "description": "OSDiskType represents the type of an OS disk on an agent pool."
+    },
+    "KubeletConfig": {
+      "description": "Kubelet configurations of agent nodes.",
+      "type": "object",
+      "properties": {
+        "cpuManagerPolicy": {
+          "type": "string",
+          "description": "CPU Manager policy to use."
+        },
+        "cpuCfsQuota": {
+          "type": "boolean",
+          "description": "Enable CPU CFS quota enforcement for containers that specify CPU limits."
+        },
+        "cpuCfsQuotaPeriod": {
+          "type": "string",
+          "description": "Sets CPU CFS quota period value."
+        },
+        "imageGcHighThreshold": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The percent of disk usage after which image garbage collection is always run."
+        },
+        "imageGcLowThreshold": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The percent of disk usage before which image garbage collection is never run."
+        },
+        "topologyManagerPolicy": {
+          "type": "string",
+          "description": "Topology Manager policy to use."
+        },
+        "allowedUnsafeSysctls": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Whitelist of unsafe sysctls or unsafe sysctl patterns (ending in `*`)."
+        },
+        "failSwapOn": {
+          "type": "boolean",
+          "description": "If set to true it will make the Kubelet fail to start if swap is enabled on the node."
+        }
+      }
+    },
+    "LinuxOSConfig": {
+      "description": "OS configurations of Linux agent nodes.",
+      "type": "object",
+      "properties": {
+        "sysctls": {
```

Comment:
since this is not an array conside renaming to singular i.e. sysctl instead of sysctls

Reply:
Cutoff date of datamodel in our service has passed for this API version. I can fix in next API version.

Reply:
change property name is breaking change, which isn't allowed even in new Api version, need to go through api board review